src/scripts/main.js

```diff
@@ -1,7 +1,95 @@
 'use strict';
 
-// Uncomment the next lines to use your game instance in the browser
-// const Game = require('../modules/Game.class');
-// const game = new Game();
+const Game = require('../modules/Game.class');
+const game = new Game();
 
-// Write your code here
+const startButton = document.querySelector('.start');
+const startMessage = document.querySelector('.message-start');
+const loseMessage = document.querySelector('.message-lose');
+const winMessage = document.querySelector('.message-win');
+const gameScore = document.querySelector('.game-score');
+
+startButton.addEventListener('click', () => {
+  if (startButton.classList.contains('start')) {
+    game.start();
+    setState();
+
+    startButton.classList = 'button restart';
```

Comment:
Directly setting `classList` to a string can be risky as it may overwrite existing classes unintentionally. Consider using `classList.add()` and `classList.remove()` to manipulate individual classes.